### PR TITLE
closes 4 change scoring in transductiveevaluator to use crossentropy instead of symmetric kl

### LINKED ISSUE #4: Change scoring in `TransductiveEvaluator` to use crossentropy instead of symmetric KL
symmetric KL does not work well when degenerate distributions are involved

A cross-entropy should work better for this purpose

### PR DESCRIPTION
**Pull request by Tremirre**
Switches to using cross entropy instead of symmetric KL for reliability evaluation.

Replaces the default merger in `TransductiveEvaluator` to numpy-compatible one (as it is most common)

### CODE REVIEW

File: merci/evaluate.py [hunk 1]
```diff
@@ -79,12 +79,44 @@ class TransductiveEvaluator(ModelEvaluator):
...
+        :param test: A tuple containing input and target data for testing
+        :return: A tuple containing the merged input and target data
+        """
+        return (
+            np.concatenate([train[0], test[0]]),
+            np.concatenate([train[1], test[1]]),
+        )
+
+    @staticmethod
+    def list_merger(
```
Comment (azywot) [inline merci/evaluate.py:99]: Do we still need `list_merger`?

**Review by azywot — APPROVED** [review]
Clear, LGTM 👍
*one minor com

**Review by zuzg — APPROVED** [review]

File: merci/evaluate.py (hunk 1 above)
Comment (Tremirre) [reply merci/evaluate.py:99]: I left it in case we want to use some classifiers operating on list objects  (or for testing or anything like that)

Comment (azywot) [reply merci/evaluate.py:99]: Okok

**Pull request merged**